src/index.html

```diff
@@ -6,8 +6,40 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <h1 class="imac"></h1>
+      <div class="text">
+        <p class="headline">
+          APPLE A1419 iMac 27" Retina
+          <br>
+          5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="code">
+          Product code: 195434
+        </p>
+
+        <div class="module">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="review">Reviews: 5</div>
+        </div>
+
+        <h2 class="price">
```

Comment:
this should be a BEM element of card block, use the proper classname according to BEM naming convention

Check yourself with the [style guide](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#bem)